src/copy-paste.js

```diff
@@ -329,27 +330,39 @@ define([
         init: function () {
             var opts = this.opts().copyPaste;
             vellum = this;
+            isMac = /Mac/.test(navigator.platform);
+
+            function copyModifierKeyPressed(event) {
+                if (isMac && event.metaKey) {
+                    return true;
+                }
+
+                return event.ctrlKey;
```

Comment:
Oops. I think that was a bug anyways. that would have returned true if it a was a mac and ctrl(command?) key was pressed.